packages/account/src/Components/account-limits/account-limits-context.ts

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+export type TAccountLimitsContext = {
+    currency?: string;
+    footer_ref?: React.ReactElement | DocumentFragment | object;
+    overlay_ref?: React.RefObject<HTMLElement>;
+    toggleOverlay?: () => boolean;
+};
+
+const AccountLimitsContext = React.createContext<TAccountLimitsContext>({
+    currency: '',
+    footer_ref: undefined,
```

Comment:
Do we need to assign the value as undefined? Can we ignore those props?

Reply:
Orite will remove the props 👌